### PR TITLE
px4flow add parameter to enable (SENS_EN_PX4FLOW)

### PR DESCRIPTION
**Pull request by dagar**
- fixes https://github.com/PX4/Firmware/issues/12571

The new parameter (SENS_EN_PX4FLOW) needed to enable the PX4 Flow needs to be mentioned in release notes (migration guide) to help existing users transition.

### CODE REVIEW

**Comment by dagar** [comment]
@hyonlim FYI

File: ROMFS/px4fmu_common/init.d/rcS [hunk 1]
```diff
@@ -457,7 +457,10 @@ else
 	fi
 
 	# Check for flow sensor, launched as a background task to scan
-	px4flow start &
+	if param compare SENS_EN_PX4FLOW 1
+	then
+		px4flow start &
```
Comment (mhkabir) [inline ROMFS/px4fmu_common/init.d/rcS]: Just curious, why do we start this in the background with &, when everything else starts regularly.

Comment (dagar) [reply ROMFS/px4fmu_common/init.d/rcS]: It spends a fairly long time (~ 5 seconds) probing (multiple attempts) in the main.

**Comment by Ken-Thorburn** [comment]
Hey, I can't find the parameter sens_en_px4flow within px4 1.9.2. If this parameter hasn't rolled out yet, I am unable to get the sensor working anyway #12911

**Pull request merged**